package.json

```diff
@@ -84,25 +84,21 @@
 	"devDependencies": {
 		"@babel/plugin-transform-react-jsx": "^7.9.1",
 		"@types/jasmine": "^3.10.3",
-		"chai": "^3.5.0",
-		"copyfiles": "^1.0.0",
+		"chai": "^4.3.7",
```

Comment:
See #453 for justification, but need to update some of these here w/ the type fix to satisfy CI.